common/windows/delphi/packages/PackageInfo.pas

```diff
@@ -780,6 +805,19 @@ procedure TPackageOptions.SetGraphicFile(const Value: TPackageContentFile);
   end;
 end;
 
+procedure TPackageOptions.SetLicenseFile(const Value: TPackageContentFile);
+begin
+  if Assigned(FLicenseFile) then FLicenseFile.RemoveNotifyObject(LicenseRemoved);
+  if not Assigned(Value) then
+    FLicenseFile := nil
+  else
+  begin
+    if Value.Package <> Package then raise EPackageInfo.CreateFmt(SGraphicNotOwnedCorrectly, [Value]);
```

Comment:
Is there an error for `SLicenseNotOwnedCorrectly`?

Reply:
Good catch.